core/cloudflow-flink/src/main/scala/cloudflow/flink/FlinkStreamlet.scala

```diff
@@ -18,22 +18,21 @@ package cloudflow.flink
 
 import java.nio.file.Path
 
-import org.slf4j.LoggerFactory
+import cloudflow.streamlets.BootstrapInfo._
+import cloudflow.streamlets._
 import com.typesafe.config.Config
-
-import scala.concurrent.{ Future, Promise }
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.util.{ Failure, Try }
+import net.ceedubs.ficus.Ficus._
 import org.apache.flink.api.common.JobExecutionResult
 import org.apache.flink.api.common.typeinfo.TypeInformation
 import org.apache.flink.client.program.OptimizerPlanEnvironment
 import org.apache.flink.streaming.api.CheckpointingMode
 import org.apache.flink.streaming.api.datastream.{ DataStreamSink, DataStream ⇒ JDataStream }
 import org.apache.flink.streaming.api.environment.CheckpointConfig
 import org.apache.flink.streaming.api.scala._
-import net.ceedubs.ficus.Ficus._
-import cloudflow.streamlets._
-import cloudflow.streamlets.BootstrapInfo._
+
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
We're using the following order on imports:
java
scala
thirdparty
cloudflow imports

Reply:
better we configure this with autofmt?

Reply:
I don't think scalafmt supports that.